# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../util-linux/0002-cfdisk-add-tinfo-libs.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

In the case libtinfo was build and installed as a separate library
cfdisk needs to be linked against it.

build error
----------------------------------------------------------------------
i686-unknown-linux-gnu/bin/ld: fdisks/cfdisk-cfdisk.o: undefined reference to symbol 'keypad'
/build/morfoh/opensde/master-musl/build/min2-master-generic-x86-pentium2/lib/libtinfow.so.5: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
----------------------------------------------------------------------

--- ./fdisks/Makemodule.am.orig	2014-01-24 15:53:13.610905311 +0100
+++ ./fdisks/Makemodule.am	2014-01-24 15:53:47.043670665 +0100
@@ -68,7 +68,7 @@
 cfdisk_LDADD += -lslang
 else
 cfdisk_CFLAGS += $(NCURSES_CFLAGS)
-cfdisk_LDADD += $(NCURSES_LIBS)
+cfdisk_LDADD += $(NCURSES_LIBS) $(TINFO_LIBS)
 endif
 endif # BUILD_CFDISK
 
